db/migrations/20230404074213_create_sub_categories_table.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE sub_categories {
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  main_cat_id INT NOT NULL,
```

Comment:
```suggestion
  main_category_id INT NOT NULL,
```
foreign key 네이밍은 가급적 축약된게 아닌 해당 table의 단수단위로 작성될 수 있게 해주세요~
동일한 맥락의 피드백이 적용될 수 있는 부분은 일괄적으로 반영될 수 있게 수정 부탁드리겠습니다 :)